dsaas-services/build-tool-detector.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+- hash: 8b49875a886a1fb2a290afb70121edac6eb2b321
```

Comment:
please bump version to recent one `3ea0b04` that exists on quay to fix ci check failure

Reply:
@sbose78

Reply:
Done :)